modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/DefaultExecutionMediator.java

```diff
@@ -90,29 +108,53 @@ public Object execute(OperationExecutor executor, OperationContext context) thro
         }
     }
 
-    private MutableConfigurationStats getMutableConfigurationStats(OperationContext context)
+    private Object executeWithRetryPolicy(OperationExecutor executor, OperationContext context, List<Interceptor> interceptors) throws Throwable
     {
-        ConfigurationStats stats = context.getConfiguration().getStatistics();
-        return stats instanceof MutableConfigurationStats
-               ? (MutableConfigurationStats) stats
-               : null;
+        if (retryPolicyTemplate == null)
+        {
+            Optional<ConnectionProvider> connectionProvider = context.getConfiguration().getConnectionProvider();
+            retryPolicyTemplate = getRetryPolicyTemplate(connectionProvider);
+            LifecycleUtils.initialiseIfNeeded(retryPolicyTemplate, muleContext);
```

Comment:
In the last review we talked about how this wasn't needed anymore and how it can cause the intialise() method been called miltiple timeso n the retryPolicy